tests/test_packageAlerts.py

```diff
@@ -0,0 +1,389 @@
+                     "dipNdata": 0,
+                     "flags": 1})
+
+    return pd.DataFrame(data=data)
+
+
+def _roundTripThroughApdb(objects, sources, dateTime):
+    """Run object and source catalogs through the Apdb to get the correct
+    table schemas.
+
+    Parameters
+    ----------
+    objects : `pandas.DataFrame`
+        Set of test DiaObjects to round trip.
+    sources : `pandas.DataFrame`
+        Set of test DiaSources to round trip.
+    dateTime : `datetime.datetime`
+        Time for the Apdb.
+
+    Returns
+    -------
+    objects : `pandas.DataFrame`
+        Round tripped objects.
+    sources : `pandas.DataFrame`
+        Round tripped sources.
+    """
+    tmpFile = tempfile.NamedTemporaryFile()
+
+    apdbConfig = ApdbConfig()
+    apdbConfig.db_url = "sqlite:///" + tmpFile.name
+    apdbConfig.isolation_level = "READ_UNCOMMITTED"
+    apdbConfig.dia_object_index = "baseline"
+    apdbConfig.dia_object_columns = []
+    apdbConfig.schema_file = _data_file_name(
+        "apdb-schema.yaml", "dax_apdb")
+    apdbConfig.column_map = _data_file_name(
+        "apdb-ap-pipe-afw-map.yaml", "ap_association")
+    apdbConfig.extra_schema_file = _data_file_name(
+        "apdb-ap-pipe-schema-extra.yaml", "ap_association")
+
+    apdb = Apdb(config=apdbConfig,
+                afw_schemas=dict(DiaObject=make_dia_object_schema(),
+                                 DiaSource=make_dia_source_schema()))
+    apdb.makeSchema()
+
+    minId = objects["pixelId"].min()
+    maxId = objects["pixelId"].max()
+    diaObjects = apdb.getDiaObjects([[minId, maxId + 1]], return_pandas=True).append(objects)
+    diaSources = apdb.getDiaSources(np.unique(objects["diaObjectId"]),
+                                    dateTime,
+                                    return_pandas=True).append(sources)
+
+    apdb.storeDiaSources(diaSources)
+    apdb.storeDiaObjects(diaObjects, dateTime)
+
+    diaObjects = apdb.getDiaObjects([[minId, maxId + 1]], return_pandas=True)
+    diaSources = apdb.getDiaSources(np.unique(diaObjects["diaObjectId"]),
+                                    dateTime,
+                                    return_pandas=True)
+    diaObjects.set_index("diaObjectId", drop=False, inplace=True)
+    diaSources.set_index(["diaObjectId", "filterName", "diaSourceId"],
+                         drop=False,
+                         inplace=True)
+
+    return (diaObjects, diaSources)
+
+
+class TestPackageAlerts(unittest.TestCase):
+
+    def setUp(self):
+        np.random.seed(1234)
+        self.center = lsst.geom.Point2D(50.1, 49.8)
+        self.bbox = lsst.geom.Box2I(lsst.geom.Point2I(-20, -30),
+                                    lsst.geom.Extent2I(140, 160))
+        self.dataset = lsst.meas.base.tests.TestDataset(self.bbox)
+        self.dataset.addSource(100000.0, self.center)
+        exposure, catalog = self.dataset.realize(
+            10.0,
+            self.dataset.makeMinimalSchema(),
+            randomSeed=0)
```

Comment:
I am lost reading this code, but I think that's just from unfamiliarity with the LSST stack. I'm fine just trusting that what you have here is right.

Reply:
Yeah, it's taken from some other test code and basically just allows me to create an exposure with random values easier.

Reply:
Maybe think about putting it in a common utils package and documenting it. But just maybe - only if it helps you.